### DISCUSSION TITLE
1 Model for Multi Filament Resources

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#6980] [Help]
Hi All,

How I can achieve, To make 1 mode (1 table) for multi-resources

![image](https://github.com/filamentphp/filament/assets/6214964/c242f49e-54ad-4a06-aea8-7423ead87963)

Every User in each department can access 1 table, and each department only can view and edit the related fields with they own department only. And they will have deferent widgets also.

Thank you

**Comment by pboivin** [comment]
It's possible to have multiple resources for the same model, all you need is to set the `$slug` property on the other resources:

```php
class ProductResource extends Resource
{
    protected static ?string $model = Product::class;
    
    // ...
```

```php
class OtherResource extends Resource
{
    protected static ?string $model = Product::class;
    
    protected static ?string $slug = 'other';
    
    // ...
```

But you should know that there are other issues that come from this, specifically around authorization. It's been discussed a lot on Discord lately, if you want to dig a bit deeper:
https://discord.com/channels/883083792112300104/1117896731183546458/1117896731183546458
